liquidity-mining/src/lib.rs

```diff
@@ -702,7 +702,10 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
         amm_pool_id: T::AmmPoolId,
         multiplier: FarmMultiplier,
     ) -> Result<YieldFarmId, DispatchError> {
-        ensure!(!multiplier.is_zero(), Error::<T, I>::InvalidMultiplier);
+        ensure!(
+            multiplier.ge(&MIN_YIELD_FARM_MULTIPLIER),
```

Comment:
nitpick: the following doesn't work?
```suggestion
            multiplier >= &MIN_YIELD_FARM_MULTIPLIER,
```